apps/web/src/locales/en/messages.json

```
@@ -6457,6 +6457,18 @@
   "billingSyncHelp": {
     "message": "Billing Sync help"
   },
+  "licensePaidFeaturesHelp": {
+    "message": "License paid features help"
+  },
+  "selfHostGracePeriodHelp": {
+    "message": "After your subscription expires, you have 60-days to apply an updated license file to your organization. Grace period ends $GRACE_PERIOD_END_DATE$.",
```

Dash/hyphen is unnecessary

```suggestion
    "message": "After your subscription expires, you have 60 days to apply an updated license file to your organization. Grace period ends $GRACE_PERIOD_END_DATE$.",
```